### DISCUSSION TITLE
connect sp-api and get orders data

### DISCUSSION THREAD

**Discussion by nomi992** [amzn/selling-partner-api-models#3544] [General]
we are trying to use amazon sp-api and get the information about reports but it gave us error "Error: [{'message': 'Access to requested resource is denied.', 'code': 'Unauthorized', 'details': ''}]" we configured roles policies and users and also register application but its still give us same error.

![Untitled-1](https://user-images.githubusercontent.com/71398761/224232101-09940807-5647-4db7-ad67-e4ce2052d4ab.jpg)
![Untitled-2](https://user-images.githubusercontent.com/71398761/224232106-08c9f74a-38b4-4233-836b-2e143765837b.jpg)
![Untitled-3](https://user-images.githubusercontent.com/71398761/224232111-83557f39-51bd-413c-8960-6bb34d85b739.jpg)
![Untitled-4](https://user-images.githubusercontent.com/71398761/224232114-aeb29fc7-574a-40e4-a7d6-1418f61e5176.jpg)
![Untitled-5](https://user-images.githubusercontent.com/71398761/224232116-a02972fd-573c-46a6-82ff-4a0b1a182d90.jpg)
![image](https://user-images.githubusercontent.com/71398761/224232245-af922790-9bd4-4225-9a78-0053a422827e.png)
![image](https://user-images.githubusercontent.com/71398761/224232283-2c074d67-e583-427f-aefc-0eca5a905b84.png)

**Comment by JonhSilver** [comment]
Probably you using wrong role_arn or not allowed this type of report in admin panel
